lib/types/host-rules.ts

```diff
@@ -14,4 +14,8 @@ export interface HostRule {
   enableHttp2?: boolean;
   concurrentRequestLimit?: number;
   resolvedHost?: string;
+  keyStore?: {
```

Comment:
This also still needs the config option defined. Check the `lib/config/options/index.ts` 

https://github.com/renovatebot/renovate/blob/main/lib/config/options/index.ts

Reply:
I'm not sure quite what'd be appropriate for `keyStore`, since it's an object with a `string|Buffer` child and an optional passphrase string.

Reply:
what kind of processing do these env and cli flags get? I could take a string which is `file path:passphrase`, read only once on startup. AFAIU, this config method is irrelevant for the hosted case, so it doesn't matter that it reads files?